src/tasks/deployment.ts

```diff
@@ -76,7 +61,8 @@ async function generateDeployment(
   const settings = {
     ...inputSettings,
     virtualCowToken: {
-      ...inputSettings.virtualCowToken,
+      gnoPrice: inputSettings.gnoPrice,
+      nativeTokenPrice: inputSettings.nativeTokenPriceOnETH,
```

Comment:
If we hardcode the native token xdai price to 1, then this could just be `ethPrice` or something like this.